Makefile.dep

```diff
@@ -175,6 +175,10 @@ ifneq (,$(filter gnrc_netif,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter gnrc_netif_manual_init,$(USEMODULE)))
+  USEMODULE += auto_init_gnrc_netif
```

Comment:
Why enforce this?

Reply:
`auto_init_gnrc_netif()` is only available with the `auto_init_gnrc_netif` module.
And manual init should overwrite auto init.